staticconf/config.py

```diff
@@ -439,6 +439,29 @@ def has_changed(self):
         return last_mtime < self.last_max_mtime
 
 
+def create_LoggingMTimeComparator(err_logger):
+    """Return an MTimeComparator subclass that calls err_logger(filename)
+    if os.path.getmtime(filename) fails."""
+
+    def try_getmtime_else_log(filename):
+        try:
+            return os.path.getmtime(filename)
+        except OSError:
+            err_logger(filename)
```

Comment:
@bchess `sys.exc_info()` should do that since the function is called within the exception context. And with `sys.exc_info()` you can also see the traceback, not just the exception object. I should probably make it clear that that's the reason the `err_logger` is called within the exception context, though.